h/browser/chrome/help/index.html

```diff
@@ -105,6 +105,13 @@ <h1 class="help-item__heading">We’re sorry, Hypothesis doesn’t work on this
         <img class="help-item__icon" src="sad-annotation.svg" />
         <h1 class="help-item__heading">We’re sorry, Hypothesis doesn't work on this site yet.</h1>
       </section>
+      <section id="other-error" class="help-item">
+        <img class="help-item__icon" src="sad-annotation.svg" />
+        <p class="center">
+          There was a problem loading Hypothesis into this page:
```

Comment:
The other pages make use of an `<h1>` which looks slightly smarter.

In addition, I think we need to gear this copy towards less technical users. So perhaps something like:

```
Oh no! There was a problem loading Hypothesis into this page.

This isn't meant to happen, and we're sorry it's happened to you. Our team has been notified of the error and will try and fix it as soon as possible. If it keeps happening, please get in touch with support@hypothes.is, and pass them these technical details of the error:

<code><pre>
...
</pre></code>
```

We are reporting these errors to Sentry, yes?